crowdkit/__init__.py

```diff
@@ -1,5 +1,5 @@
 from typing import List
 
-__version__ = '1.2.1'
+__version__ = "1.2.1"
```

Comment:
Here and in other changed lines: I personally prefer `'` to `"` and this is probably not standardized. Can we keep the single quotes by default?

Reply:
Here's an explanation from Black: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings

It has some fair points about why double quotes are better and if we just switch to the black codestyle, we standartize everything about our codestyle and everyone would be able to follow it effortlessly because black will automatically reformat the code through pre-commit.

So, I suggest to standartize Crowd-Kit to use double quotes with this PR.

Reply:
Thank you for sharing your perspective and the link to the Black code style guidelines. I've taken the time to review the information provided, and I respectfully agree with your suggestion to standardize Crowd-Kit to use double quotes. It makes sense, especially considering the benefits of having a consistent code style that can be effortlessly maintained through pre-commit.

I appreciate your diligence in maintaining code quality, and I'm fully supportive of this proposal. Let's proceed with the change to double quotes in this PR.

Thank you again for your contribution and attention to detail.